autoupgrade.php

```diff
@@ -79,7 +79,11 @@ public function install()
             $tab = new Tab();
             $tab->class_name = 'AdminSelfUpgrade';
             $tab->module = 'autoupgrade';
-            $tab->id_parent = (int) Tab::getIdFromClassName('AdminTools');
+
+            // We use 'DEFAULT' tab class to be compatible from v1.7 to v9 but we can that's appear in the "more" part of the menu
+            // 'DEFAULT' is a reference to deprecated tabs
```

Comment:
```suggestion
            // We use DEFAULT to add Upgrade tab as a standalone tab in the back office menu
```

Reply:
@ga-devfront one comment + I changed title of the PR :)

Reply:
done